runtime/gc_glue_java/ObjectModel.cpp

```diff
@@ -119,7 +120,11 @@ GC_ObjectModel::internalClassLoadHook(J9HookInterface** hook, UDATA eventNum, vo
 			clazz->classDepthAndFlags |= J9AccClassGCSpecial;
 			objectModel->_classClass = clazz;
 		} else if (0 == compareUTF8Length(J9UTF8_DATA(className), J9UTF8_LENGTH(className), (U_8*)abstractOwnableSynchronizer, strlen(abstractOwnableSynchronizer))) {
-			 clazz->classDepthAndFlags |= J9AccClassOwnableSynchronizer;
+			clazz->classDepthAndFlags |= J9AccClassOwnableSynchronizer;
+		} else if (0 == compareUTF8Length(J9UTF8_DATA(className), J9UTF8_LENGTH(className), (U_8*)continuation, strlen(continuation))) {
```

Comment:
you need to check if the class is `jdk/internal/vm/Continuation/` or a subclass

Reply:
might be easier if you look for `jdk/internal/vm/Continuation` and save it as `_continuationClass` then add a `isSameOrSuperClassOf` test to set the bit. If its too difficult to do in GC code, we can do it in ramClass creation

Reply:
https://hyc-runtimes-jenkins.swg-devops.com/view/OpenJ9%20-%20Personal/job/Pipeline-Build-Test-Personal/13972/